packages/clay-labels-input-field/src/ClayLabelsInputField.soy

```diff
@@ -0,0 +1,171 @@
+{namespace ClayLabelsInputField}
+
+/**
+ * This renders the component's whole content.
+ */
+{template .render}
+	{@param helpText: string}
+	{@param spritemap: string}
+	{@param? _handleButtonClicked: any}
+	{@param? _handleCloseButtonClick: any}
+	{@param? _handleDropdownItemClick: any}
+	{@param? _handleOnInput: any}
+	{@param? _handleOnKeydown: any}
+	{@param? _removeFocusedLabel: any}
+	{@param? contentRenderer: string}
+	{@param? elementClasses: string}
+	{@param? filteredItems: list<?>}
+	{@param? id: string}
+	{@param? label: string}
+	{@param? selectedLabels: list<?>}
+
+	{let $attributes kind="attributes"}
+		class="from-group
+			{if $elementClasses}
+				{sp}{$elementClasses}
+			{/if}
+		"
+
+		{if $id}
+			id="{$id}"
+		{/if}
+	{/let}
+
+	<div {$attributes}>
+		{if $label}
+			<label>{$label}</label>
+		{/if}
+
+		{call .content}
+			{param _handleButtonClicked: $_handleButtonClicked /}
+			{param _handleCloseButtonClick: $_handleCloseButtonClick /}
+			{param _handleDropdownItemClick: $_handleDropdownItemClick /}
+			{param _handleOnInput: $_handleOnInput /}
+			{param _handleOnKeydown: $_handleOnKeydown /}
+			{param _removeFocusedLabel: $_removeFocusedLabel /}
+			{param contentRenderer: $contentRenderer /}
+			{param filteredItems: $filteredItems /}
+			{param helpText: $helpText /}
+			{param selectedLabels: $selectedLabels /}
+			{param spritemap: $spritemap /}
+		{/call}
+	</div>
+{/template}
+
+{template .content}
+	{@param helpText: string}
+	{@param spritemap: string}
+	{@param? _handleButtonClicked: any}
+	{@param? _handleCloseButtonClick: any}
+	{@param? _handleDropdownItemClick: any}
+	{@param? _handleOnInput: any}
+	{@param? _handleOnKeydown: any}
+	{@param? _removeFocusedLabel: any}
+	{@param? contentRenderer: string}
+	{@param? filteredItems: list<?>}
+	{@param? selectedLabels: list<?>}
+
+	<div class="input-group input-group-stacked-sm-down">
+		<div class="input-group-item">
+			<div data-keydown="{$_handleOnKeydown}" ref="formGroupInput" class="form-control dropdown-full form-control-tag-group">
+				{if $selectedLabels}
```

Comment:
For easier integration, it might be good to have hidden input fields here with the label values. That way the component will be compatible with standard form submission.

We can't just select labels, they need to have values associated with them, so our input data should look more like:

```
data: [{
    value: '123',
    label: 'Element 1'
}, {
    value: '456',
    label: 'Element 2'
}]
```

On the UI, we should show the labels, but we would be updating and keeping a list of `<input type="hidden">` with the values of the selected items. There should also be a way to configure the name of the input so the server will be able to retrieve the data as a native array as expected.